docker/e2e/scenarios/smoke.sh

```
@@ -0,0 +1,86 @@
+case "$header" in
+  *"age-encryption.org"*|*"BEGIN AGE ENCRYPTED FILE"*)
+    pass "artifact $sample_artifact is age-encrypted (header: $header)" ;;
+  *)
+    fail "artifact $sample_artifact is NOT age-encrypted (header: $header)" ;;
+esac
+
+step "CRITICAL: vault must NOT contain plaintext of the stub credential"
+# Walk every blob in the vault and grep for the stub key. Even one match = leak.
+stub_key="${OPENAI_API_KEY:-sk-stub-for-bootstrap}"
+leak=$(git --git-dir=/vault/repo.git rev-list --objects --all | awk '{print $1}' | sort -u | \
+       while read sha; do
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)